pyformance/__version__.py

```diff
@@ -1 +1 @@
-__version__ = "2.0.0"
+__version__ = "2.0.1"
```

Comment:
Why?

Reply:
What do you mean? it's a new release of Pyformance, I incremented the version...